pkg/core/transaction/witness_scope.go

```diff
@@ -61,8 +64,8 @@ func ScopesFromString(s string) (WitnessScope, error) {
 // scopesToString converts witness scope to it's string representation. It uses
 // `, ` to separate scope names.
 func scopesToString(scopes WitnessScope) string {
-	if scopes == Global {
-		return "Global"
+	if scopes&Global != 0 || scopes == FeeOnly {
```

Comment:
Why not `== Global` ?

Reply:
And this method is not supposed to return an error, so just Global if it was detected.